Framework/Core/src/PropertyTreeHelpers.cxx

```diff
@@ -28,39 +28,41 @@ void PropertyTreeHelpers::populate(std::vector<ConfigParamSpec> const& schema, b
   for (auto& spec : schema) {
     // strip short version to get the correct key
     std::string key = spec.name.substr(0, spec.name.find(","));
-    try {
-      switch (spec.type) {
-        case VariantType::Int:
-          pt.put(key, vmap[key].as<int>());
-          break;
-        case VariantType::Int64:
-          pt.put(key, vmap[key].as<int64_t>());
-          break;
-        case VariantType::Float:
-          pt.put(key, vmap[key].as<float>());
-          break;
-        case VariantType::Double:
-          pt.put(key, vmap[key].as<double>());
-          break;
-        case VariantType::String:
-          if (auto v = boost::any_cast<std::string>(&vmap[key].value())) {
-            pt.put(key, *v);
-          }
-          break;
-        case VariantType::Bool:
-          pt.put(key, vmap[key].as<bool>());
-          break;
-        case VariantType::Unknown:
-        case VariantType::Empty:
-        default:
-          throw std::runtime_error("Unknown variant type");
+    if (vmap.count(key) > 0) {
```

Comment:
why not simply:

```
if (vmap.count(key) == 0) {
  continue;
}
```

I would prefer if our code remains with the least amount of indentation level possible.

Reply:
Done.

Reply:
are you sure about the > ?